app/routes/__index/dns-records/new.tsx

```diff
@@ -47,15 +47,16 @@ export const action = async ({ request }: ActionArgs) => {
 
 export default function NewDnsRecordRoute() {
   const actionData = useActionData();
+  logger.info(actionData);
```

Comment:
Sorry, is this needed?

Reply:
Sorry, I tried to add a dynamic text and see if what's in this object. I should remove this.